index.js

```diff
@@ -441,6 +441,21 @@
   //. ```
   Maybe.prototype.of = def('Maybe#of', [a], Maybe.of);
 
+  //# Maybe#reduce :: Maybe a ~> (b -> a -> b) -> b -> b
+  //.
+  //. Takes a function and an initial value, returning:
+  //. * the initial value if this is a Nothing; otherwise
+  //. * the result of applying the function to the initial value and the value
+  //.   of the Just.
```

Comment:
Two last minor points:

  - in the first list item, "this" should be wrapped in backticks; and
  - in the second list item, do you think we should write "the value of <del>the</del> <ins>this</ins> Just"?

I'm sorry for nitpicking, but I do consider the wording of these descriptions important.